test/csv/parse/test_row_separator.rb

```diff
@@ -13,4 +13,17 @@ def test_multiple_characters
                    CSV.parse("a\r\nb\r\n", row_sep: "\r\n"))
     end
   end
+
+  def test_skip_row_recovery_after_faulty_quoting_with_row_ending_with_cr_lf
```

Comment:
This file is for `row_sep` option. Could you move this to `test/csv/parse/test_invalid.rb`?

We can simplify test name like `test_ignore_invalid_line_cr_lf`.